packages/contracts-bedrock/justfile

```diff
@@ -29,6 +29,10 @@ test-kontrol-no-build:
 test-rerun: build-go-ffi
   forge test --rerun -vvv
 
+# Run extra fuzz iterations for new fuzz tests.
+test-extra-fuzzy: build-go-ffi
```

Comment:
we should be mindful of fun rather than straightforward names, can make it difficult to onboard new people, especially esl people

Reply:
Updated to `test-extra-fuzz-runs.sh`

Reply:
Changed to `test-heavy-fuzz-new-tests.sh` for clarity